lib/orders.rb

```diff
@@ -0,0 +1,41 @@
+require "order"
+
+class Orders
+  attr_reader :orders
+
+  def initialize
+    @orders = []
+  end
+
+  def add_order(order)
+    @orders << order
```

Comment:
Is it possible for the `order` to be empty?

Reply:
not currently, but you are right nil protection here is probably a good idea